include/realizations/catchment/Formulation_Constructors.hpp

```diff
@@ -27,6 +31,10 @@ namespace realization {
         {"tshirt", create_formulation_constructor<Tshirt_Realization>()},
         {"tshirt_c", create_formulation_constructor<Tshirt_C_Realization>()},
         {"simple_lumped", create_formulation_constructor<Simple_Lumped_Model_Realization>()}
+#ifdef NGEN_LSTM_TORCH_LIB_ACTIVE
```

Comment:
This is  the correct use of macro configuration,  but  I think we are going to want  to think a bit about  the long term impacts of using this feature.  @robertbartel  any thoughts from  you perspective on using optional libraries/realizations in this way?  My initial concern is in complicating  these  constructors/functions  for someone  trying to add  something  new.  Just something to think about for now.